dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/InternalConfigs.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.sbt
+
+import sbt._
+
+object InternalConfigs {
+  private[sbt] val devModeConfig = config("lagom-dev").hide
+  private[sbt] val cassandraDevModeConfig = config("lagom-dev-cassandra").hide
```

Comment:
Hey James, really good points, thanks for the feedback. Scoping dev dependencies to `Runtime` makes a lot of sense (btw, do you see any benefit in introducing a new configuration that extends Runtime? Because I don't.), and works almost out-of-the-box. The "almost" is due to the `cassandra-registration` module, as I need to inspect the project's classpath to decide if it should be injected. This is why I had two different configurations, and I'm not seeing yet how I can achieve the same result without using `managedClasspath in cassandraDevModeConfig`.